layouts/_default/baseof.html

```diff
@@ -25,5 +25,7 @@
         {{ block "footer" . }}{{ partial "layout/footer.html" . }}{{ end }} {{ block "scripts" . }}{{ partial "layout/scripts.html" . }}{{ end }}
       </div>
     </div>
+    <!-- 100% privacy-first analytics -->
+    <script async src="https://scripts.simpleanalyticscdn.com/latest.js"></script>
```

Comment:
We might add it here: https://github.com/gnoverse/gnops/blob/main/layouts/partials/layout/scripts.html :)

Reply:
I didn't catch we had a dedicated scripts injection, nice.

Moved:

[96a6ebe](https://github.com/gnoverse/gnops/pull/19/commits/96a6ebed235232e56ab7bb4973126e872012c685)